e2e/cypress/support/ui_commands.js

```diff
@@ -432,3 +432,10 @@ Cypress.Commands.add('checkRunLDAPSync', () => {
         }
     });
 });
+
+Cypress.Commands.add('archiveCurrentChannel', () => {
```

Comment:
oh! I think we both have a ui command for archive current channel, we might need to remove one at some point (but they are almost the same, [different placement though](https://github.com/mattermost/mattermost-webapp/pull/6358/files#diff-08c1f6e408556f0b68412a28840bda2aR46))
/cc @saturninoabril

Reply:
Yeah, I think the one you have is what we'll use moving forward. So if it gets merged first, then I'd suggest to update and use it in this PR 😄

Reply:
Changed it to use the other method